tests/chainer_tests/dataset_tests/test_download.py

```diff
@@ -121,14 +121,14 @@ def tearDown(self):
     def test_fail_to_make_dir(self):
         with mock.patch('os.makedirs') as f:
             f.side_effect = OSError()
-            with self.assertRaises(RuntimeError):
+            with self.assertRaises(OSError):
```

Comment:
As this PR does not change the behavior of `cached_download`, we need not to change the type of the error.